wellcomeml/datasets/winer.py

```diff
@@ -0,0 +1,222 @@
+# WiNER: A Wikipedia Annotated Corpus for Named Entity Recognition
+# https://www.aclweb.org/anthology/I17-1042/
+
+from wellcomeml.datasets.download import check_cache_and_download
+
+# def load_winer():
+import tarfile
+import os
+from tqdm import tqdm 
+import json
+import random
+
+def create_train_test(
```

Comment:
is this the function that takes the source data and transform them to train and test? could that be mentioned more explicitly in the comment section below as this is a function that someone might need to use, if say we want to recreate the files since you do not store them in the raw format.